en/django_templates/README.md

```diff
@@ -89,7 +89,7 @@ $ git push
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ cd my-first-blog
+$ cd djangogirls
```

Comment:
I think it should be `USERNAME.pythonanywhere.com` when using the deployment script. I guess `$USER.pythonanywhere.com` will work as well.